cmd/derper/derper.go

```diff
@@ -317,14 +325,45 @@ func main() {
 		}
 		err = rateLimitedListenAndServeTLS(httpsrv)
 	} else {
-		log.Printf("derper: serving on %s", *addr)
+		log.Printf("derper: serving on %s (pid %d)", *addr, os.Getpid())
 		err = httpsrv.ListenAndServe()
 	}
 	if err != nil && err != http.ErrServerClosed {
 		log.Fatalf("derper: %v", err)
 	}
 }
 
+// runNiceDERPChild starts a child process at a slightly lower priority that
+// does everything except run the STUN server.
+//
+// This means that STUN packets get answered by a separate, higher-priority
+// process.
+func runNiceDERPChild() {
+	// Build an arg list, filtering out the --stun flag.
+	var args []string
+	flag.Visit(func(f *flag.Flag) {
+		if f.Name == "stun" {
+			return
+		}
+		args = append(args, "-"+f.Name, f.Value.String())
```

Comment:
If we have (or add) any Boolean flags, I think we will want `-name=value` format as the blanks will not be read properly and will terminate option parsing.